src/lando/main/tests/__init__.py

```diff
@@ -66,3 +66,22 @@ def test_require_phabricator_api_key(monkeypatch, optional, valid_key, status):
         assert resp.body.api_token == "custom-key"
 
     assert resp.status_code == status
+
+
+@pytest.mark.django_db(transaction=True)
+def test_phabricator_api_key_encryption():
+    user = User.objects.create_user(username="test_user", password="test_password")
+    profile = Profile.objects.create(user=user)
+
+    assert profile.phabricator_api_key == ""
+
+    # Set an arbitrary key.
+    key = "test-key"
+    profile.save_phabricator_api_key(key)
+    assert profile.encrypted_phabricator_api_key
+    assert profile.phabricator_api_key == key
+
+    # Clear the key
+    profile.encrypted_phabricator_api_key = b""
+    profile.save()
+    assert profile.phabricator_api_key == ""
```

Comment:
We could codify this, it is performed [here](https://github.com/mozilla-conduit/lando/pull/130/files#diff-51c5ef361c4f25928f715e8bc2cb8bafa5fbb9602fcd1ab5632b313c7d04f559R21-R22).

I don't necessarily see a harm in encrypting the cleared value, though I don't see a big advantage either (other than possibly obfuscating which values are actual keys vs. not in case there is a data breach, but then we have a bigger problem). Will do this though.

Reply:
Making direct changes to the encrypted value is fine, I'd just prefer we have a single method that we use to clear the Phab API key for profiles.